hexrd/unitcell.py

```diff
@@ -1982,17 +1982,42 @@ def atom_pos(self):
 
     @atom_pos.setter
     def atom_pos(self, val):
+        """
+        SS 03/08/2021 fixing some issues with
+        updating asymmetric positions after 
+        updating atominfo
+        fixing 
+        """
+        if hasattr(self, 'atom_type'):
+            if self.atom_ntype != val.shape[0]:
+                raise ValueError(f"incorrect number of atom positions."
+                                 "number of atom type = {self.atom_ntype} "
+                                 "and number of"
+                                 "atom positions = {val.shape[0]}.")
```

Comment:
Each string needs to start with "f" for it to be an f-string. So:

```python
                raise ValueError(f"incorrect number of atom positions. "
                                 f"number of atom type = {self.atom_ntype} "
                                 f"and number of "
                                 f"atom positions = {val.shape[0]}.")
```

Be sure to also add spaces at the end of each string so it won't come out like `"positions.number of"`